src/components/pages/public/RegisterPage.vue

```diff
@@ -1,12 +1,69 @@
 <template>
-  <div>
-    This is the register page.
-    <router-view></router-view>
+  <div class="register-page">
+    <div class="register-page__wrapper">
+      <h1 class="register-page__heading">Ahoy you!</h1>
+      <h2 class="register-page__subheading">Care to register?</h2>
+      <RegisterFirstCard />
+    </div>
   </div>
 </template>
 
 <script>
-export default {};
+import RegisterFirstCard from "@/components/RegisterFirstCard.vue";
+
+export default {
+  name: "RegisterFirstPage",
```

Comment:
there are two files with almost the same content `RegisterPage.vue` and `RegisterFirstPage.vue`